backend/dataall/modules/datapipelines/services/datapipelines_permissions.py

```diff
@@ -0,0 +1,50 @@
+from dataall.db.permissions import ENVIRONMENT_INVITED, ENVIRONMENT_INVITATION_REQUEST, \
+    ENVIRONMENT_ALL, TENANT_ALL, TENANT_ALL_WITH_DESC, RESOURCES_ALL, RESOURCES_ALL_WITH_DESC
+
+
+"""
+DATAPIPELINE PERMISSIONS FOR ENVIRONMENT
+"""
+CREATE_PIPELINE = 'CREATE_PIPELINE'
+LIST_PIPELINES = 'LIST_PIPELINES'
+
+ENVIRONMENT_INVITED.append(CREATE_PIPELINE)
+ENVIRONMENT_INVITED.append(LIST_PIPELINES)
```

Comment:
`LIST_PIPELINES` is not used anywhere, but here. I think we are safe to delete from the code and the DB